app/controllers/assignments_controller.rb

```diff
@@ -1,5 +1,5 @@
 class AssignmentsController < ApplicationController
-  before_action :set_assignment, only: [:show, :edit, :update, :destroy]
+  before_action :set_assignment, only: [:show, :edit, :update, :destroy, :delete_reminder]
```

Comment:
You don't need to add `:delete_reminder` here since it's just a method, not a controller action (there's no route for it).